doc/source/overview/algorithms.md

```diff
@@ -6,38 +6,38 @@ The following tables summarize the advised use cases for the current algorithms.
 
```

Comment:
Mention that the *Feature Level* column is also relevant for drift detection. As discussed we can improve the tables further in a followup PR.